lib/tasks/docker.rake

```diff
@@ -91,10 +95,10 @@ task "docker:test:setup" do
   setup_test_env(
     setup_multisite: true,
     create_db: true,
-    create_parallel_db: false,
-    load_plugins: false,
-    install_all_official: false,
-    update_all_plugins: false,
+    create_parallel_dbs: !!ENV["CREATE_PARALLEL_DATABASES"],
```

Comment:
Hmm wouldn't !!0 be true...

Reply:
It would be but we don't support any other values other than `1` here.

Reply:
Yeah I guess it's not obvious that 0 is not accepted. I've set `QUNIT_EMBER_CLI=0` assuming it turns it off so I guess that's wrong now 😅